# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tetex/glibc.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur tetex-src-3.0-orig/texk/dvipsk/afm2tfm.c tetex-src-3.0/texk/dvipsk/afm2tfm.c
--- tetex-src-3.0-orig/texk/dvipsk/afm2tfm.c	2009-08-22 00:54:14.000000000 +0000
+++ tetex-src-3.0/texk/dvipsk/afm2tfm.c	2009-08-22 01:14:07.000000000 +0000
@@ -257,7 +257,7 @@
 }
 
 int
-getline P1H(void) {
+getLine P1H(void) {
    register char *p ;
    register int c ;
 
@@ -606,7 +606,7 @@
    ai = newchar() ;
    ai->adobenum = -1 ;
    ai->adobename = "||" ; /* boundary character name */
-   while (getline()) {
+   while (getLine()) {
       switch(interest(paramstring())) {
 case FontName:
          fontname = paramnewstring() ;
@@ -1882,7 +1882,7 @@
 
    while (1) {
       while (param == 0 || *param == 0) {
-         if (getline() == 0)
+         if (getLine() == 0)
             error("! premature end in encoding file") ;
          for (p=buffer; *p; p++)
             if (*p == '%') {
@@ -1973,7 +1973,7 @@
       p = gettoken() ;
       if (strcmp(p, "]"))
          error("! token 258 in encoding must be make-array (])") ;
-      while (getline()) {
+      while (getLine()) {
          for (p=buffer; *p; p++)
             if (*p == '%') {
                if (ignoreligkern == 0)
diff -Nur tetex-src-3.0-orig/texk/web2c/cpascal.h tetex-src-3.0/texk/web2c/cpascal.h
--- tetex-src-3.0-orig/texk/web2c/cpascal.h	2009-08-22 00:54:14.000000000 +0000
+++ tetex-src-3.0/texk/web2c/cpascal.h	2009-08-22 01:12:32.000000000 +0000
@@ -241,6 +241,12 @@
 #define getname vms_getname
 #endif
 
+
+#ifdef getline
+#undef getline
+#endif
+#define getline web2c_getline
+
 /* Declarations for the routines we provide ourselves in lib/.  */
 
 extern string basenamechangesuffix P3H(const_string,const_string,const_string);
